lib/eslint-patch.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var meld = require('meld');
+var docs = require('./docs');
+
+var supportedPlugins = ['react', 'babel'];
+
+module.exports = function patcher(eslint) {
+
+  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
This patch is not working as intended. It was at the time it was written, but (as you noted) that's no longer the case given the current internals of ESLint.

We're evaluating if we want to accept unsupported plugins causing errors like that, or try to bring back the logic intended to replace that with ignoring them. Unfortunately, this `meld` approach doesn't appear viable on the new internals, so it would require moving back to our own fork of ESLint itself.

In the meantime though, we should probably remove this.